includes/class-sensei-course.php

```diff
@@ -2671,13 +2671,14 @@ public static function course_archive_sorting( $query ) {
 		$course_order_by_options = apply_filters(
 			'sensei_archive_course_order_by_options',
 			array(
+				'default' => __( 'Sort by default', 'sensei-lms' ),
```

Comment:
I think I'd probably name this "Default sort" even though it doesn't quite match the naming of the other items. It changes the meaning a bit to call it "Sort by default". To me, it's kinda like saying "just sort this page by default". But it's already going to sort this page by default regardless of whether an option is explicitly selected.

Whereas "Default sort" implies some other custom order will be used, although the student won't really know (or care) what that custom order is.

Reply:
Fair enough! Changing 🚀